server/engine/src/moco_pass/templates/template_moco.py.txt

```diff
@@ -0,0 +1,104 @@
+import opensim as osim
+import matplotlib; matplotlib.use('Agg')

```

Comment:
I understand this backend for matplotlib is to write results into file instead of real time rendering. I've seen it in a few files. Is there any way of setting this as a global option, or in a configuration file, so it does not have to be repeated in every file?

Reply:
We could provide [a `matplotlibrc` file](https://matplotlib.org/1.4.2/users/customizing.html) as an alternative, and one option is to put it in the working directory. This might be slightly overkill just for this setting, and some of the other usages of `matplotlib.use('Agg')` are removed in this PR. 

It is a TXT file because it is not a valid Python script: it is just a template. A copy gets made and filled in as a valide Python during the `moco_pass` pipeline.

Reply:
Another downside of using a `matplotlibrc` is that it would have to be copied over into the OpenSim results folder so that users can run the resulting .py file from a different directory. For that reason, I think it's simpler, at least for now, to just leave the backend call in the script.

Reply:
I think that clarifies it. I agree it is better for now to keep the backend call.